dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -36,13 +36,17 @@ warning.could.not.thin.application.explanation=The application could not be conv
 warning.could.not.thin.application.useraction=Attempt to resolve the issue in the exception message and either restart the server or update the application. 
 
 error.need.springboot.version.15=CWWKC0252E: Could not run the application because the wrong version of the Spring Boot feature is configured in server.xml. The application requires the feature springBoot-1.5 to be configured.
-error.need.springboot.version.15.explanation= The application requires Spring Boot version 1.5 but a wrong version of the Spring Boot feature is configured in the server.xml.  
-error.need.springboot.version.15.useraction= The springBoot-1.5 feature needs to be configured in the server.xml.
+error.need.springboot.version.15.explanation=The application requires Spring Boot version 1.5 but a wrong version of the Spring Boot feature is configured in the server.xml.  
+error.need.springboot.version.15.useraction=The springBoot-1.5 feature needs to be configured in the server.xml.
 
 error.need.springboot.version.20=CWWKC0253E: Could not run the application because the wrong version of the Spring Boot feature is configured in server.xml. The application requires the feature springBoot-2.0 to be configured.
-error.need.springboot.version.20.explanation= The application requires Spring Boot version 2.0 but a wrong version of the Spring Boot feature is configured in the server.xml.  
-error.need.springboot.version.20.useraction= The springBoot-2.0 feature needs to be configured in the server.xml.
+error.need.springboot.version.20.explanation=The application requires Spring Boot version 2.0 but a wrong version of the Spring Boot feature is configured in the server.xml.  
+error.need.springboot.version.20.useraction=The springBoot-2.0 feature needs to be configured in the server.xml.
 
-error.missing.servlet=CWWKC0254E: Could not run the application because it requires a Servlet feature to be configured in the server.xml. Either the servlet-3.1 or servlet-4.0 need be configured.
-error.missing.servlet.explanation= The application is a web application and it requires a Servlet feature to be configured in the server.xml.
-error.missing.servlet.useraction= Either the  servlet-3.1 or servlet-4.0 need be configured in the server.xml.
+error.missing.servlet=CWWKC0254E: Could not run the application because it requires a Servlet feature to be configured in the server.xml. For example, the servlet-4.0 feature can be configured.
+error.missing.servlet.explanation=The application is a web application and it requires a Servlet feature to be configured in the server.xml.
+error.missing.servlet.useraction=A Servlet feature needs to be configured in the server.xml. For example, the servlet-4.0 feature can be configured.
```

Comment:
+error.missing.servlet - A quick question about the missing servlet messages. Can **any** servlet be configured? Or does this message need to refer to a specific servlet? Does the user need to know which servlet to configure, or can it be anything?

Reply:
It depends on the application being deployed.  In almost all cases it will not matter, servlet-3.1 or servlet-4.0 can be used.  But it is possible they have a Spring Boot application that is dependent on a specific version of Servlet.

Reply:
Okay, I think I'm starting to understand. The web application requires a Servlet feature configured, and some MIGHT require a specific servlet, but most won't. I think that the "For example" part is a little confusing-- is it really necessary? How do users find what their options are for servlets that would resolve this problem? 

+error.missing.servlet=CWWKC0254E: Could not run the application because it requires a Servlet feature to be configured in the server.xml file. 
+error.missing.servlet.explanation=The application is a web application and it requires a Servlet feature to be configured in the server.xml file. 
+error.missing.servlet.useraction=A Servlet feature needs to be configured in the server.xml file.

Reply:
It probably is not necessary to have the "For example".  I'm fine with your suggested change.

@anjumfatima90, please make the requested message updates.  Thanks.